warp10/src/main/java/io/warp10/plugins/tcp/TCPManager.java

```diff
@@ -51,6 +51,7 @@
   private static final int DEFAULT_QSIZE = 1024;
   private static final int DEFAULT_MAXMESSAGES = 1;
   private static final int DEFAULT_MAXCONNECTIONS = 1;
+  private static final int DEFAULT_TCP_BACKLOG = 128;
```

Comment:
The plugin was configured with 50 as an arbitrary value, it is better to use 0 as default, like for the other backlogs.